### PR TITLE
add: ability to import a key pair

### PR DESCRIPTION
**Pull request by sirpy**
format of keypair should be an object {publicKey:hexstring, privateKey:hexstring}

### CODE REVIEW

**Comment by aphelionz** [comment]
@sirpy Thank you for adding this, and thanks for your patience as i just got around to this now. 

Might you be able to add some tests?

File: src/keystore.js [hunk 1]
```diff
@@ -96,6 +96,20 @@ class Keystore {
 
     return keys
   }
+  
+  async importKey(id,keyPair) {
+     if(await this.hasKey(id))
+     {
```
Comment (Evalir) [inline src/keystore.js:102]: This brace should probably be moved up 😄

**Review by Evalir — COMMENTED** [review]
Was browsing the code and saw this open PR 😄

**Comment by aphelionz** [comment]
It's still a valid PR! I think just needs doc and tests

**Comment by aphelionz** [comment]
Closing this for now, stale and needs tests.

**Pull request closed without merging**